filters/ThirdParty/filter_252_DandelionSproutSerbian/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "filterId": 252,
+  "name": "Dandelion Sprout's Serbo-Croatian List",
+  "description": "For the first time in known history, this is a filterlist for websites in Serbian, Montenegrin, Croatian and Bosnian. This means that all significant language groups in Europe now have filterlists for them.",
+  "timeAdded": 1647852134737,
+  "homepage": "https://github.com/DandelionSprout/adfilt",
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 7,
+  "subscriptionUrl": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/SerboCroatianList.txt",
+  "tags": [
+    "purpose:ads",
+    "lang:sr"
```

Comment:
We'll need `lang:hr` as well.

Also, don't forget that the tag definition must be added as well.
Please try building the registry to see if there're missing tags.